app/js/App.js

```diff
@@ -52,12 +52,11 @@ class App {
 
   createPage() {
     const page = new Page()
-    page.setTheme(this.theme)
+    page.setTheme((this.settings) ? this.settings.theme : 'default')
```

Comment:
No need to use a ternary operator here since `this.settings` will always be a `Settings` object, and it has a default theme already